uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequestCalendarFeedConfig.php

```diff
@@ -271,6 +264,26 @@ public function addSelectWhereClause() {
     return $clauses;
   }
 
+  /**
+   * Gets the ids of the feeds that are not accessible to the current user from the
+   * list of feed configs passed in.
+   *
+   * @param array $limitedVisibilityFeedConfigs
+   *
+   * @return array
+   */
+  private function getInAccessibleFeedConfigsForCurrentUser($limitedVisibilityFeedConfigs) {
```

Comment:
Two small things here, maybe change the name to `getInAccessibleFeedConfigIDsForCurrentUser` and you can just 

```
return $this->getInAccessibleFeedConfigs(...);
```

Instead of creating a new variable